backend/controller/controller.go

```diff
@@ -721,14 +721,14 @@ func (s *Service) CreateDeployment(ctx context.Context, req *connect.Request[ftl
 	}
 
 	ingressRoutes := extractIngressRoutingEntries(req.Msg)
-	dname, err := s.dal.CreateDeployment(ctx, ms.Runtime.Language, module, artefacts, ingressRoutes)
+	dkey, err := s.dal.CreateDeployment(ctx, ms.Runtime.Language, module, artefacts, ingressRoutes, nil, time.Now())
```

Comment:
cronjob service will be asked for cron jobs based on the module schema here, but this is skipped as part of this smaller PR.
Full version here: https://github.com/TBD54566975/ftl/blob/matt2e%2Fcron-jobs/backend/controller/controller.go#L740